### ISSUE TITLE
E13 Theme for E23/E24

### ISSUE THREAD

**Issue by mathieujobin** [mbert/etheme-vulcan-retro-e20#1]
They there, thank you for making the Vulcan theme..

beautiful and it inspired me to ask you...

I know this is the wrong place to ask
but would you be able to the E13 remake for E16 and port it to E23+ ? 

https://github.com/mathieujobin/themes/tree/master/e13/e16

I got the full E13 source if that is useful

**Comment by mbert** [comment]
I'm afraid I cannot help here. My contribution to the work published here is hardly more than making a few adaptations on something that was already mostly e20-ready. I would love to see more of the old themes ported to e20, like ShinyMetal or SpiffE, but that work would need to be done pretty much from the scratch, and I am neither qualified nor do I have the time for this.
I actually got pretty frustrated with the very low-level theming API that is very beginner-unfriendly, as I commented in the linuxquestions forum: https://www.linuxquestions.org/questions/bodhi-92/enlightenment-0-23-1-in-bl5-testing-repo-4175661585/#post6070356
There are some (far too few) people into e theming amongst the Bodhi folks, so you might try to get in contact with them.
